packages/server/lib/errors.js

```diff
@@ -923,6 +923,11 @@ const getMsgByType = function (type, arg1 = {}, arg2, arg3) {
         Enable write permissions to this directory to ensure screenshots and videos are stored.
 
         If you don't require screenshots or videos to be stored you can safely ignore this warning.`
+    case 'EXPERIMENTAL_SAMESITE_REMOVED':
+      return stripIndent`\
+        The \`experimentalGetCookiesSameSite\` configuration option was removed in Cypress 5. Yielding the \`sameSite\` property is now the default behavior of the \`cy.cookie\` commands.
```

Comment:
I think the convention when listing versions is to say `in Cypress version x.x.x`

Reply:
This was the only prior art I could find, which also isn't that:

https://github.com/cypress-io/cypress/blob/c00f6cdac2fddaeeabb8ee4bd4c62adbbfd8ea04/packages/server/lib/errors.js#L140-L141

Reply:
In our migration guide we usually say Cypress 5, so as not to be restricted of it only applying to Cypress 5.0.0 and not to 5.1.0, etc.